lib/printer.go

```diff
@@ -22,25 +22,26 @@ type DuplexVendorMap map[cdd.DuplexType]string
-	Manufacturer       string                         // CUPS: PPD;                         GCP: manufacturer field
-	Model              string                         // CUPS: PPD;                         GCP: model field
-	GCPVersion         string                         //                                    GCP: gcpVersion field
-	SetupURL           string                         //                                    GCP: setup_url field
-	SupportURL         string                         //                                    GCP: support_url field
-	UpdateURL          string                         //                                    GCP: update_url field
-	ConnectorVersion   string                         //                                    GCP: firmware field
-	State              *cdd.PrinterStateSection       // CUPS: various;                     GCP: semantic_state field
-	Description        *cdd.PrinterDescriptionSection // CUPS: translated PPD;              GCP: capabilities field
-	CapsHash           string                         // CUPS: hash of PPD;                 GCP: capsHash field
-	Tags               map[string]string              // CUPS: all printer attributes;      GCP: repeated tag field
-	DuplexMap          DuplexVendorMap                // CUPS: PPD;
-	NativeJobSemaphore *Semaphore
-	QuotaEnabled       bool
-	DailyQuota         int
+	GCPID               string                         //                                    GCP: printerid (GCP key)
+	Name                string                         // CUPS: cups_dest_t.name (CUPS key); GCP: name field
+	DefaultDisplayName  string                         // CUPS: printer-info;                GCP: default_display_name field
+	UUID                string                         // CUPS: printer-uuid;                GCP: uuid field
+	Manufacturer        string                         // CUPS: PPD;                         GCP: manufacturer field
+	Model               string                         // CUPS: PPD;                         GCP: model field
+	GCPVersion          string                         //                                    GCP: gcpVersion field
+	SetupURL            string                         //                                    GCP: setup_url field
+	SupportURL          string                         //                                    GCP: support_url field
+	UpdateURL           string                         //                                    GCP: update_url field
+	ConnectorVersion    string                         //                                    GCP: firmware field
+	State               *cdd.PrinterStateSection       // CUPS: various;                     GCP: semantic_state field
+	Description         *cdd.PrinterDescriptionSection // CUPS: translated PPD;              GCP: capabilities field
+	CapsHash            string                         // CUPS: hash of PPD;                 GCP: capsHash field
```

Comment:
looks like you added a bunch of whitespace before "GCP:" in this line. Mistake?

Reply:
they are auto formatted.